apps/prairielearn/elements/pl-figure/pl-figure.mustache

```diff
@@ -1,8 +1,8 @@
 {{^inline}}
-<div class="container-fluid">
-    <img src="{{src}}" {{#width}}width="{{width}}"{{/width}} alt="{{alt}}" class="img-fluid mx-auto d-block" />
+<div class="container-fluid {{#class}}{{class}}{{/class}}" title="{{title}}">
+    <img src="{{src}}" {{#width}}width="{{width}}"{{/width}} alt="{{alt}}"  class="img-fluid mx-auto d-block" />
```

Comment:
Any reason why this is set in the `div` instead of the image in this case?

Reply:
In testing, when the `title` attribute is on the `img` tag, the tooltip wouldnt appear. It seems the `container-fluid` is overriding certain attributes. Same thing when adding certain Bootstrap classes like `border`.